build/commands/lib/logging.js

```diff
@@ -21,7 +21,7 @@ process.stdout.on('resize', setLineLength)
 const progressStyle = chalk.bold.inverse
 const statusStyle = chalk.green.italic
 const warningStyle = chalk.black.bgYellow
-const errorStyle = chalk.black.bgRed
+const errorStyle = chalk.black.red
```

Comment:
This doesn't make much sense. Should just be `chalk.red` if that's the style you want.